packages/teams/src/utils/get-allowed-group-access.ts

```diff
@@ -10,10 +11,15 @@ import { AGOAccess } from "../types";
  */
 export function getAllowedGroupAccess(
   requestedAccess: AGOAccess,
-  user: IUser
+  user: IUser,
+  portal: IPortal
 ): AGOAccess {
-  // figure out what they can create...
-  const canCreatePublic = hasAllPrivileges(user, GROUP_ACCESS_PRIVS.public);
+  // portal-wide flag takes presidence, and is not sync'd with privs
+  const portalWideCanSharePublic = portal.canSharePublic;
```

Comment:
in order to prevent a breaking change, if portal is not passed in we should assume it's allowed, right?